typings/custom-scope-defs.d.ts

```diff
@@ -45,6 +45,8 @@ interface ICustomScope extends ng.IScope {
     showDropArea?: boolean;
     getRecorderController?: (() => void);
     openAddAudioTranslationModal?: ((files: any) => void);
+    userIsGuest?: boolean;
```

Comment:
This and the next don't seem like they should be custom defs, since they are defined entirely in Oppia (right)?

Reply:
I am sorry but I did not get your query. Any property added to scope in link function should be added here to avoid `property does not exist on type ng.IScope` error.

Reply:
Sorry, I might have misunderstood this one. Could you please add comments/docs at the top of the file to explain what it is used for and when to put something in it?

Reply:
Done.